packages/@vuepress/theme-default/src/client/components/global/CodeGroup.ts

```diff
@@ -63,12 +64,37 @@ export default defineComponent({
                 h(
                   'button',
                   {
+                    ref: (element) => {
+                      if (element) {
+                        tabRefs.value[i] = element as HTMLButtonElement
+                      }
+                    },
                     class: `code-group__nav-tab${
                       i === activeIndex.value
                         ? ' code-group__nav-tab-active'
                         : ''
                     }`,
+                    ariaPressed: i === activeIndex.value,
+                    ariaExpanded: i === activeIndex.value,
```

Comment:
`i === activeIndex.value` expression has been used 3 times here. We'd better extract it to a variable